jetstream/experimenter.py

```diff
@@ -170,11 +170,18 @@ class ExperimentCollection:
     # for nimbus experiments
     EXPERIMENTER_API_URL_V6 = "https://experimenter.services.mozilla.com/api/v6/experiments/"
 
+    # experiments that are in draft state
+    EXPERIMENTER_API_URL_DRAFTS = (
```

Comment:
Suggestion because we still have Normandy and V1, and I often forget about them.
```suggestion
    # nimbus experiments that are in draft state
    EXPERIMENTER_API_URL_V6_DRAFTS = (
```